### PR TITLE
Link to our official Gitter chat room

### PR DESCRIPTION
**Pull request by clue**
We've deprecated our old IRC channel and now use a Gitter chat room
instead: https://gitter.im/reactphp/reactphp

Refs #429
Refs https://github.com/reactphp/website/pull/50

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -2,10 +2,11 @@
     "name": "react/react",
     "description": "ReactPHP: Event-driven, non-blocking I/O with PHP.",
     "keywords": ["reactor", "asynchronous", "ReactPHP"],
+    "homepage": "https://reactphp.org/",
     "license": "MIT",
     "support": {
         "issues": "https://github.com/reactphp/react/issues",
-        "irc": "irc://irc.freenode.org/reactphp"
+        "chat": "https://gitter.im/reactphp/reactphp"
```
Comment (WyriHaximus) [inline composer.json]: Should we add this to all our packages?

Comment (clue) [reply composer.json]: I'm undecided, but I can see some use for this :man_shrugging:

Comment (WyriHaximus) [reply composer.json]: It shows up on the packagist page for each package so might not be a bad idea 😄

Comment (clue) [reply composer.json]: @WyriHaximus My vote would be to postpone this discussion until we see we're happy with adoption of the new chat room and then discuss this in a separate ticket in a few weeks and not hold this one back :shipit:

Comment (WyriHaximus) [reply composer.json]: @clue of  course, just thinking out loud :)

**Pull request merged**